Bio/SeqUtils/ProtParam.py

```diff
@@ -4,8 +4,11 @@
 """Simple protein analysis.
 
 Example::
-
-    X = ProteinAnalysis("MAEGEITTFTALTEKFNLPPGNYKKPKLLYCSNGGHFLRILPDGTVDGTRDRSDQHIQLQLSAESVGEVYIKSTETGQYLAMDTSGLLYGSQTPSEECLFLERLEENHYNTYTSKKHAEKNWFVGLKKNGSCKRGPRTHYGQKAILFLPLPV")
+    protein = '''
+    MAEGEITTFTALTEKFNLPPGNYKKPKLLYCSNGGHFLRILPDGTVDGTRDRSDQHIQLQLSAESVGEVYIKSTE
+    TGQYLAMDTSGLLYGSQTPSEECLFLERLEENHYNTYTSKKHAEKNWFVGLKKNGSCKRGPRTHYGQKAILFLPL
+    PV'''
```

Comment:
That gives a different string - it would have new line characters embedded in it. The implicit line concatenation would be better if you really want to avoid the long line.

Since you are working on this, can you make it into a doctest too?

Reply:
Yes.